docs/server-side-configuration.md

```diff
@@ -0,0 +1,37 @@
+
+---
+
+# Cozy MyAccounts: Server side configuration
+
+## Authorized categories list
+
+The MyAccounts categories names list won't be automatically computed from available connectors in order to keep it clear and relevant for all Cozy users with a growing number of connectors.
+
+Instead, they need to be configured on the served side, in a simple way, from a config file. This is a list of categories authorized to be used:
+
+```javascript
+{
+    authorizedCategories:
+        {
+            'insurance': 1,
```

Comment:
Great @nono! Thanks for these explanations. I completely forgot the object prototype properties here and to use hasOwnProperty, my bad :/. Finally, it's not readable as before.

Also agree with the coffescript usage here on server side. 

I just think about performance here because the category checking will be processed for each konnector each time the client will request it. Of course, it's not so much important for us here with few konnectors and few categories.